wsd/COOLWSD.cpp

```diff
@@ -2340,19 +2340,19 @@ void COOLWSD::innerInitialize(Application& self)
         TraceDumper = Util::make_unique<TraceFileWriter>(path, recordOutgoing, compress, takeSnapshot, filters);
     }
 
+    WebServerPoll = Util::make_unique<TerminatingPoll>("websrv_poll");
+
+    PrisonerPoll = Util::make_unique<PrisonPoll>();
+
+    Server = Util::make_unique<COOLWSDServer>();
+
 #if !MOBILEAPP
     SavedClipboards = Util::make_unique<ClipboardCache>();
 
     LOG_TRC("Initialize FileServerRequestHandler");
     FileServerRequestHandler::initialize();
 #endif
 
-    WebServerPoll = Util::make_unique<TerminatingPoll>("websrv_poll");
```

Comment:
Not so sure on this one - one of the things I like from a security perspective is having loaded all the files from the file-system, and having cached them internally before we start our various other polls on listening sockets and accept external messages.
Perhaps we should have a large comment // ---------------- from here on we start getting external messages ----------------
in there in the original place.
If we need to have another part of this doing later can we have it in a method after webserver init etc. which should happen after the fileserver setup.